client/src/lib/error-telemetry.js

```diff
@@ -15,27 +15,42 @@ class ErrorTelemetry {
     this.options = options;
   }
 
+  authenticate(uuid, token) {
+    logger.info(`Error telemetry: uuid=${uuid}, token=${token}`);
```

Comment:
Please remove this log statement. I don't believe it's adding any value at this point